src/components/NavBar/NavbarMobile.jsx

```diff
@@ -57,9 +57,7 @@ const NavbarMobile = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+          <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo' aria-label='logo'/>
```

Comment:
see other comment, but do styling inline for now. we are trying to get rid of `styles.css` file.

Reply:
hey scott, i believed i made all required changes.